src/token/hToken.sol

```diff
@@ -188,4 +188,11 @@ contract hToken is ERC20H {
   function updateSupportedWrapper(address token, bool supported) external onlyHolographer onlyOwner {
     _supportedWrappers[token] = supported;
   }
+
+  /**
+   * @dev Temporarily placed to bypass bytecode conflicts
+   */
+  function isHToken() external pure returns (bool) {
```

Comment:
When will we remove this given the bycode conflict?

Reply:
This was temporarily added so that I could run extensive onchain tests on top of the regular localhost tests. Will be removed when we are ready to change deployment salt.